src/niscope/system_tests/test_system_niscope.py

```diff
@@ -195,6 +195,34 @@ def test_fetch_double_into(session):
             assert record_wfm[j] == waveform[i * test_record_length + j]
 
 
+def test_get_self_cal_last_date_time(session):
+    last_cal = session.get_self_cal_last_date_and_time()
+    assert last_cal.month == 12
+    assert last_cal.day == 21
+    assert last_cal.year == 1999
+    assert last_cal.hour == 0
+    assert last_cal.minute == 0
+
+
+def test_get_ext_cal_last_date_time(session):
+    last_cal = session.get_ext_cal_last_date_and_time()
+    assert last_cal.month == 12
+    assert last_cal.day == 21
+    assert last_cal.year == 1999
+    assert last_cal.hour == 0
+    assert last_cal.minute == 0
```

Comment:
just in case don't use two zeroes (won't catch swapped values)

Reply:
I was under the impression that the simulated devices had a fixed expected output for the datetime, as specified in the comments for nidmm's equivalent system test. Both nidmm and nidcpower use 0 for hour and minute. Is there somewhere that the simulated datetime can actually be changed?

Reply:
No, you're right. In my brain I was thinking unit tests and you can configure the mock to return whatever you want. But these are system tests.